File: Java/Program/SistemOperasi.java
public class SistemOperasi {
    private String nama;
    private String versi;
    private String berlisensi;

    public SistemOperasi(String nama, String versi, String berlisensi) {
        this.nama = nama;
        this.versi = versi;
        this.berlisensi = berlisensi;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getNama() {
        return nama;
    }

    public void setVersi(String versi) {
        this.versi = versi;
    }

    public String getVersi() {
        return versi;
    }

    public void setBerlisensi(String berlisensi) {
        this.berlisensi = berlisensi;
    }
    
    public String getBerlisensi() {
        return berlisensi;
    }
}

File: CPP/Program/SistemOperasi.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class SistemOperasi
{
private:
    string nama;
    string versi;
    string berlisensi;

public:
    SistemOperasi()
    {
    }

    SistemOperasi(string nama, string versi, string berlisensi)
    {
        this->nama = nama;
        this->versi = versi;
        this->berlisensi = berlisensi;
    }

    void setNama(string nama)
    {
        this->nama = nama;
    }

    void setVersi(string versi)
    {
        this->versi = versi;
    }

    void setBerlisensi(bool berlisensi)
    {
        this->berlisensi = berlisensi;
    }

    string getNama()
    {
        return this->nama;
    }

    string getVersi()
    {
        return this->versi;
    }

    string getBerlisensi()
    {
        return this->berlisensi;
    }

    ~SistemOperasi()
    {
    }
};
